### PR TITLE
add pslist_sessions plugin

### PR DESCRIPTION
**Pull request by atcuno**

### CODE REVIEW

File: volatility/framework/plugins/mac/pslist_sessions.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```
Comment (ikelos) [inline volatility/framework/plugins/mac/pslist_sessions.py:1]: New files need to have the current year in the copyright, I think?

File: volatility/framework/plugins/mac/pslist_sessions.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import logging
+from typing import Callable, Dict, Iterable
+
+from volatility.framework import constants, interfaces, objects, contexts, exceptions
+from volatility.plugins.mac import pslist
+
+vollog = logging.getLogger(__name__)
+
+class Pslist_Sessions(pslist.PsList):
+    """Lists the processes present in a particular mac memory image."""
```
Comment (ikelos) [inline volatility/framework/plugins/mac/pslist_sessions.py:14]: Docstring needs to change

File: volatility/framework/plugins/mac/pslist_sessions.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        proc_array = kernel.object(object_type = "array",
+                                   offset = sesshashtbl, 
+                                   count = table_size + 1,  
+                                   subtype = kernel.get_type("sesshashhead"))
+
+        for proc_list in proc_array:
+            # test the validity of the current element
+            # it is expected that many won't be initialized
+            try:
+                p = proc_list.lh_first
```
Comment (ikelos) [inline volatility/framework/plugins/mac/pslist_sessions.py:52]: If the exception fires, `p` won't exist and you'll get an exception further down...

**Comment by ikelos** [comment]
I've updated this to follow the new style of pslist methods.  Please check it and make sure you're happy with it @atcuno ...  5:)

**Comment by atcuno** [comment]
@ikelos I used both https://github.com/volatilityfoundation/volatility3/pull/240 and this one across some samples and both look good. Please merge both.

**Pull request merged**